checkbox-ng/plainbox/test_provider_manager.py

```diff
@@ -288,12 +291,20 @@ def test_develop(self, mock_path_entry):
         self.tool.main(["develop"])
         self.assertFileContent(filename, content)
 
+        # PROVIDERPATH defined
+        mock_getenv.return_value = "/tmp/com.example.test.provider"
+        self.tool.main(["develop"])
+        self.assertFileContent(filename, content)
```

Comment:
Is this test is wrong? 
You are checking the content of filename but provider path should be prioritized. 
It asserts that the file exists because it has been created in the same tests without a provider path.

Reply:
Stopped being lazy, splitted the test into its own function